cloudbuild.yaml

```diff
@@ -82,6 +82,8 @@ steps:
     'BRANCH_NAME=$BRANCH_NAME',
     '--env',
     'TAG_NAME=$TAG_NAME',
+    '--env',
+    'POETRY_INSTALL_ARG="--no-dev"'
```

Comment:
This argument needs to go in the 'build-image' step that starts on line 39 rather than this deployment step. You'll want to pass it with the --build-arg flag instead of --env.